.github/workflows/_checkmarx.yml

```
@@ -0,0 +1,68 @@
+name: Perform Checkmarx Scan
+
+on:
+  workflow_call: 
+
+    inputs:
+      incremental:
+        description: "Perform incremental scan"
+        type: boolean
+        default: ${{ contains(github.event_name, 'pull_request') }}
+      project_name:
+        description: "Project name in Checkmarx"
+        type: string
+        default: ${{ github.repository }}
+      upload_sarif:
+        description: "Upload SARIF output to GitHub Security Center"
```

```suggestion
        description: "Upload SARIF output to GitHub"
```
It's "Advanced Security" but I'd avoid naming it altogether.